lib/octicons_react/README.md

```diff
@@ -62,6 +62,70 @@ export default () => (
 )
 ```
 
+### `aria-labelledby`
+
+You have the option of adding accessibility information to the icon with the
+[`aria-labelledby` attribute][aria-labelledby] via the `aria-labelledby` prop. Using aria-labelledby referencing the id values of the title element provides the accessible name.
+
+```js
+// Example usage
+import {PlusIcon} from '@primer/octicons-react'
+
+export default () => (
+  <button>
+    <PlusIcon aria-labelledby="title" title="Add new item" /> New
+  </button>
+)
+```
+
+### `title`
+
+You have the option of adding accessibility information to the icon with the
+[`title` attribute][title] via the `title` prop.
+
+```js
+// Example usage
+import {PlusIcon} from '@primer/octicons-react'
+
+export default () => (
+  <button>
+    <PlusIcon title="Add new item" /> New
```

Comment:
I think this example may lead to a problematic accessible name for the `button` where it would combine the title "Add new item" with "New". Is there a better example that we could reference for usage with `title`?

Reply:
(We just removed the example all together)